src/pages/components/CertificateViewDetails.js

```diff
@@ -0,0 +1,215 @@
+                      </IconWrap>
+                      <Text>{start.split('T')[1].slice(0, 5)}</Text>
+                    </HalfWrap>
+                  </WrapDiv>
+                </GrowingDiv>
+                <CertificateTimeInterval
+                  sTimestamp={start}
+                  eTimestamp={end}
+                  bgColor="white"
+                />
+                <GrowingDiv>
+                  <HeadingDiv>End timestamp of energy use</HeadingDiv>
+                  <WrapDiv>
+                    <HalfWrap>
+                      <IconWrap>
+                        <IconDate></IconDate>Date
+                      </IconWrap>
+                      <Text>{end.split('T')[0].split('-').join('/')}</Text>
+                    </HalfWrap>
+                    <HalfWrap>
+                      <IconWrap>
+                        <IconTime></IconTime>Time
+                      </IconWrap>
+                      <Text>{end.split('T')[1].split('.')[0]}</Text>
+                    </HalfWrap>
+                  </WrapDiv>
+                </GrowingDiv>
+              </ContainerFlexNoWrapDiv>
+            </FlexLargeDiv>
+          )}
+          {energy && (
+            <FlexDiv>
+              <ContainerFullWidthWrapDiv>
+                <HeadingDiv>Electric energy use</HeadingDiv>
+                <WrapShortDiv>{energy / 1000000} kWh</WrapShortDiv>
+              </ContainerFullWidthWrapDiv>
+            </FlexDiv>
+          )}
+          {size && (
+            <FlexDiv>
+              <ContainerFullWidthWrapDiv>
+                <HeadingDiv>H2 batch size</HeadingDiv>
+                <WrapShortDiv>{size / 1000000} kWh</WrapShortDiv>
+              </ContainerFullWidthWrapDiv>
+            </FlexDiv>
+          )}
+          {eco2 && (
+            <FlexRoundedDiv>
+              <ContainerFullWidthWrapDiv>
+                <HeadingDiv>Carbon Embodiment</HeadingDiv>
+                <WrapPaddedDiv>{eco2} g CO2e</WrapPaddedDiv>
+              </ContainerFullWidthWrapDiv>
+            </FlexRoundedDiv>
+          )}
+        </ContainerFlexWrapDiv>
+      </PaddedWrapperDiv>
+    </>
+  )
+}
+
+const PaddedWrapperDiv = styled.div`
+  padding: 4px;
+  height: calc(100% - 200px);
+  min-height: 400px;
+
+  text-align: left;
+`
+
+const ContainerFlexWrapDiv = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`
+
+const FlexDiv = styled.div`
+  margin: 0px 8px 20px 8px;
+  min-width: 169px;
+  flex-grow: 1;
+
+  color: #27847a;
+  border-left: 1px solid #27847a;
+`
+
+const FlexRoundedDiv = styled.div`
+  margin: 0px 8px 20px 8px;
+  min-width: 169px;
+  flex-grow: 1;
+  padding: 20px;
+
+  border-radius: 10px;
+  background: #33e58c;
+`
+
+const FlexLargeDiv = styled(FlexDiv)`
+  min-width: 354px;
+  border-left: 0;
+`
+
+const ContainerFlexNoWrapDiv = styled.div`
+  padding-left: 5px;
+  display: flex;
+  flex-wrap: nowrap;
+`
+
+const ContainerFullWidthWrapDiv = styled.div`
+  padding-left: 5px;
+  width: 100%;
```

Comment:
div's are block elements for this `width` shouldn't be needed. It should auto to `100%`